File: 2.0/python/orders/getOrder.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.orders.getOrder(
    id=379506003,
    # return only specific fields
    fields='status,totalCost'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/orders/getOrder.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getOrder();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "status,totalCost"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
